ocs_ci/ocs/registry.py

```diff
@@ -333,3 +333,26 @@ def check_image_exists_in_registry(image_url):
         return_value = True
         logger.info("Image exists in Registry")
     return return_value
+
+
+def remove_ocp_registry_from_ocs(platform):
+    """
+    Function removes OCS registry from OCP cluster
+
+    Args:
+        platform (str): the platform the cluster deployed on
+
+    """
+    image_registry_obj = ocp.OCP(
+        kind=constants.CONFIG, namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
+    )
+    param_cmd = ''
+    if platform == 'AWS':
+        param_cmd = '[{"op": "remove", "path": "/spec/storage"}]'
+
+    elif platform == 'VMWARE':
+        param_cmd = '[{"op": "replace", "path": "/spec/storage", "value": {"emptyDir": "{}"}}]'
```

Comment:
please add also an `else` condition to handle other platforms

Reply:
something like "not supported yet?" or just pass on this step?